packages/fast-components-react-base/src/divider/divider.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+import {DividerRole, IDividerProps} from './divider.props';
+import {IDividerClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+class Divider extends Foundation<IDividerProps & IManagedClasses<IDividerClassNameContract>, React.AllHTMLAttributes<HTMLElement>, {}> {
+
+    protected handledProps: HandledProps<IDividerProps & IManagedClasses<IDividerClassNameContract>> = {
+        managedClasses: void 0,
+        role: void 0
+    };
+
+    /**
+     * Define default props
+     */
+    protected defaultProps: IDividerProps = {
+        role: void(0)
```

Comment:
You don't need `defaultProps` in this instance - we don't need to set all defaultProps to `void(0)` anymore. Or, you could also default to `DividerRole.separater` if that is the logical default.

Reply:
DividerRole.separator is intrinsically set as part of HTML5 so there default is null.  I can remove. I also, do like the new name.